protocol/messenger.go

```diff
@@ -3735,11 +3746,11 @@ func (m *Messenger) handleImportedMessages(messagesToHandle map[transport.Filter
 
 	importMessagesToSave := messageState.Response.DiscordMessages()
 	if len(importMessagesToSave) > 0 {
-		m.communitiesManager.LogStdout(fmt.Sprintf("saving %d discord messages", len(importMessagesToSave)))
+		m.logger.Debug(fmt.Sprintf("saving %d discord messages", len(importMessagesToSave)))
```

Comment:
Since it's a pure call to `zap.Logger`:
```suggestion
		m.logger.Debug("saving discord messages", zap.Int("count", len(importMessagesToSave)))
```


And a few similar calls below

Reply:
Ah yeah, that's true. Ok I can do that.

Reply:
https://github.com/status-im/status-go/pull/5257/commits/1b8a05cd1b2018285a99202baafbfa7887b2f540